var/spack/repos/builtin/packages/py-saga-python/package.py

```diff
@@ -3,17 +3,9 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
+from spack.pkg.builtin.py_radical_saga import PyRadicalSaga
 
 
-class PySagaPython(PythonPackage):
-    """A light-weight access layer for distributed computing infrastructure"""
-
-    homepage = "http://radical.rutgers.edu"
-    pypi = "saga-python/saga-python-0.41.3.tar.gz"
-
-    version('0.41.3', sha256='b30961e634f32f6008e292aa1fe40560f257d5294b0cda95baac1cf5391feb5d')
-
-    depends_on('py-setuptools',      type='build')
-    depends_on('py-apache-libcloud', type=('build', 'run'))
-    depends_on('py-radical-utils',   type=('build', 'run'))
+class PySagaPython(PyRadicalSaga):
+    """Obsolete name (alias), preserved for consistency."""
```

Comment:
The way we normally deprecate packages is to keep the package the same, but change the docstring to note that it is deprecated and add `deprecated=True` to all versions.

Reply:
This is the case when package was renamed, and could be kept as alias only (PyPI doesn't have `saga-python` anymore)

Reply:
Was `saga-python` renamed to `radical-saga`, or are they different?

Reply:
correct `saga-python`-> `radical-saga` and PyPI contains `radical-saga` from version `0.59.0`,
also can confirm we don't have this version (`0.41.3`) within github packages

Reply:
Hmm, if the package is no longer available on PyPI, and the release can't be downloaded from anywhere else, maybe we should just delete the package. Clearly no one is relying on this if it hasn't existed for some time.